### ISSUE TITLE
Fast progress, thumbs up

### ISSUE THREAD

**Issue by shanmu-raoyunfei** [hughlv/agentok#127]
Fast progress, thumbs up！Is the demo version on the official website consistent with the progress?

**Comment by hughlv** [comment]
yes. each git commit triggers auto-deployment to studio.agentok.ai.
